tests/integration/mount_propagation.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+  teardown_busybox
+  setup_busybox
+
+  # Create a "shared" parent mount point for container
+  # rootfs and make sure pivot_root() still works. This
+  cd ${BUSYBOX_BUNDLE}
+  mount --bind . .
```

Comment:
Yes, what I'm saying is to remove this from the `setup` code (where it doesn't belong because it's changing how other tests are running) and modify your `@test` so that it does something like:

```
% tmpscript=$(mktemp)
% cat >$tmpscript <<EOF
#!/bin/sh

cd ${BUSYBOX_BUNDLE}
mount --bind . .
mount --make-private .
mount --make-shared .
runc ...
EOF
% unshare -m sh $tmpscript
% rm $tmpscript
```

Which _might_ work (though it means we couldn't use the `runc` aliases I have in `helpers.bash`.